### PR TITLE
License manga update and Manga Fox Title Update

### PR DESCRIPTION
**Pull request by CarlosEsco**
fixes license status for manga fox that was not being set.
does not show chapters if manga is licensed this will prevent false issues being open for manga fox, since when licensed chapters are viewable.

Also address #926

### CODE REVIEW

**Comment by CarlosEsco** [comment]
image

**Comment by inorichi** [comment]
Can you include the mangafox chapters' title in this PR? Also fix the missing spaces etc here.

**Comment by CarlosEsco** [comment]
Merged everything into this one pull request.   Fixed spacing let me know if its still not right.  If so maybe share your java code style for android studio or which schema you are using.

File: app/src/main/java/eu/kanade/tachiyomi/source/online/HttpSource.kt [hunk 1]
```diff
@@ -197,16 +197,21 @@ abstract class HttpSource : CatalogueSource {
...
-                }
+        if (manga.status != SManga.LICENSED) {
+            return client.newCall(chapterListRequest(manga))
+                    .asObservableSuccess()
+                    .map { response ->
+                        chapterListParse(response)
+                    }
+        } else {
+            val chapters: List<SChapter> = emptyList()
+            return Observable.just(chapters)
```
Comment (inorichi) [inline app/src/main/java/eu/kanade/tachiyomi/source/online/HttpSource.kt]: Probably better to throw an exception? `Observable.error(Exception("Licensed"))` for example

Comment (CarlosEsco) [reply app/src/main/java/eu/kanade/tachiyomi/source/online/HttpSource.kt]: `return Observable.error(Exception("Licensed - No chapters to show"))`

Added a little more detail

**Comment by inorichi** [comment]
Having a `.warning` class doesn't mean the manga is licensed. I'm reverting the changes before 0.6.2

**Comment by CarlosEsco** [comment]
do you have a mangafox example where warning wasn't licensed?

**Comment by inorichi** [comment]
Any 18+ rated manga, like [Berserk](http://mangafox.me/manga/berserk).

**Comment by CarlosEsco** [comment]
k I will look into a better solution and create a new pull request

**Pull request merged**